packages/core/src/components/Table/TableRow/TableRow.tsx

```diff
@@ -101,24 +104,16 @@ export const HvTableRow = forwardRef<HTMLElement, HvTableRowProps>(
 
     const TableRow = useMemo(() => StyledTableRow(Component), [Component]);
 
-    let stripedColorEven = checkValidHexColorValue(even)
-      ? hexToRgbA(even, 0.6)
-      : even;
-
-    let stripedColorOdd = checkValidHexColorValue(odd)
-      ? hexToRgbA(odd, 0.6)
-      : odd;
+    const stripedColorEven = useRef<string>();
```

Comment:
We are still having some problems with this code:

https://github.com/lumada-design/hv-uikit-react/assets/43220251/2b0523e7-b5a6-48f4-9845-b24b777df756

The background color is not set on the first render and it doesn't update correctly.

Potential solution:

```suggestion
const [stripedColorEven, setStripedColorEven] = useState<
      string | undefined
    >();
const [stripedColorOdd, setStripedColorOdd] = useState<
      string | undefined
    >();

useEffect(() => {
      const getColor = (value?: string) => {
        if (value?.includes("var(--uikit-colors-")) {
          const color = value
            .replace("var(--uikit-colors-", "")
            .replace(")", "")
            .trim();

          return activeTheme?.colors.modes[selectedMode][color];
        } else {
          return value;
        }
};

const even = getColor(activeTheme?.table.rowStripedBackgroundColorEven);
const odd = getColor(activeTheme?.table.rowStripedBackgroundColorOdd);

setStripedColorEven(getStripedColor(even));
setStripedColorOdd(getStripedColor(odd));
}, [selectedMode, activeTheme]);

return (
      <TableRow
        ref={externalRef}
        className={cx(
          tableSectionContext.filterClassName,
          className,
          classes.root,
          classes?.[type],
          {
            [classes.hover]: hover,
            [classes.selected]: selected,
            [classes.expanded]: expanded,
            [classes.striped]: striped,
            [classes.variantList]: isList && type === "body",
            [classes.variantListHead]: isList && type === "head",
          }
        )}
        role={Component === defaultComponent ? null : "row"}
        $striped={striped}
        $stripedColorEven={stripedColorEven}
        $stripedColorOdd={stripedColorOdd}
        {...others}
      />
 );
```